<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cassatHistogram_Dialog</class>
 <widget class="QDialog" name="cassatHistogram_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Histogram功能测试界面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images/img.qrc">
    <normaloff>:/new/prefix1/icon_1.png</normaloff>:/new/prefix1/icon_1.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,10">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>98</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>Histogram功能测试</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>228</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Vangogh</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="10,1">
       <item row="0" column="0">
        <widget class="QCustomPlot" name="vangoghHistogram_widget" native="true"/>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>TDC序号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="TDC_number_comboBox">
           <item>
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="read_vangogh_Histogram_pushButton">
           <property name="text">
            <string>读取直方图</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveVangogh_histogram_pushButton">
           <property name="text">
            <string>保存图像</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filePath_lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="saveData_pushButton">
           <property name="text">
            <string>保存数据</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>248</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Cassatt</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="10,1">
       <item row="0" column="0">
        <widget class="QCustomPlot" name="cassatHistogram_widget" native="true"/>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="saveCassat_histogram_pushButton">
           <property name="text">
            <string>保存图像</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>158</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>4300</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5" rowstretch="10,1">
       <item row="0" column="0">
        <widget class="QCustomPlot" name="VI4300Histogram_widget" native="true"/>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QPushButton" name="save4300_histogram_pushButton">
           <property name="text">
            <string>保存图像</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>278</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images/img.qrc"/>
 </resources>
 <connections/>
</ui>
